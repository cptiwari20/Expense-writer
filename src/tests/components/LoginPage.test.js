import React from 'react';
import { shallow } from 'enzyme';
import loginPage, { LoginPage } from '../../components/LoginPage';

describe('Login Page', () => {
  it('should render a loginPage Component', () => {
    const wrapper = shallow(<LoginPage />)
    expect(wrapper).toMatchSnapshot()
  })
});