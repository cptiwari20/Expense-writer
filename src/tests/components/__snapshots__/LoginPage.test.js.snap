// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Page should render a loginPage Component 1`] = `
<div>
  <button>
     Login With Google 
  </button>
</div>
`;
